src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/BillingAccountAccessStep.ts

```diff
@@ -0,0 +1,43 @@
+import { h } from 'react-hyperscript-helpers'
+import { Ajax } from 'src/libs/ajax'
+import Events from 'src/libs/events'
+import { LabeledRadioButton, LabeledRadioGroup } from 'src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/LabeledRadioButton'
+import { Step } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/Step'
+import { StepFieldLegend, StepFields } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepFields'
+import { StepHeader } from 'src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader'
+
+
+export interface BillingAccountAccessStepProps {
+  isActive: boolean
+  accessToBillingAccount?: boolean
+  setAccessToBillingAccount: (boolean) => void
+}
+
+export const BillingAccountAccessStep = ({ isActive, ...props }: BillingAccountAccessStepProps) => h(Step, { isActive }, [
+  h(StepHeader, { title: 'STEP 2', children: ['Select an existing billing account or create a new one.'] }),
```

Comment:
It would be better if the legend could be `Select an existing billing account or create a new one.` vs the explanatory text below, as the radio buttons really go with `Select an existing billing account or create a new one.`.